modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -68,10 +178,34 @@ $ oc create -f bmh.yaml
 .Example output
 [source,terminal]
 ----
-secret/worker1-bmc created
-baremetalhost.metal3.io/worker1 created
+secret/openshift-worker-<num>-network-config-secret created
+secret/openshift-worker-<num>-bmc-secret created
+baremetalhost.metal3.io/openshift-worker-<num> created
 ----
++
+Where `<num>` is the worker number.
 
+. Power on and inspect the bare-metal node:
++
+[source,terminal]
+----
+$ oc -n openshift-machine-api get bmh openshift-worker-<num>
+----
++
+Where `<num>` is the worker node number.
++
+.Example output
+[source,terminal]
+----
+NAME                    STATE       CONSUMER   ONLINE   ERROR
+openshift-worker-<num>  available              true
+----
++
+[NOTE]
+====
+To allow the worker node to join the cluster, scale the `machineset` object to the number of the `BareMetalHost` objects. You can scale nodes either manually or automatically. To scale nodes automatically, use the `metal3.io/autoscale-to-hosts` annotation for `machineset`.
```

Comment:
This note is not applicable  in UPI cluster, as there are no Machineset to scale.

Reply:
Updated As Suggested.